pkg/webhook/request.go

```diff
@@ -28,21 +28,21 @@ import (
 	"github.com/banzaicloud/koperator/pkg/util"
 
 	"github.com/banzaicloud/koperator/api/v1alpha1"
+	"github.com/banzaicloud/koperator/api/v1beta1"
 )
 
 var (
-	kafkaTopic = reflect.TypeOf(v1alpha1.KafkaTopic{}).Name()
+	kafkaTopic   = reflect.TypeOf(v1alpha1.KafkaTopic{}).Name()
+	kafkaCluster = reflect.TypeOf(v1beta1.KafkaCluster{}).Name()
 )
 
 func (s *webhookServer) validate(ar *admissionv1.AdmissionReview) *admissionv1.AdmissionResponse {
 	req := ar.Request
-
 	l := log.WithValues("kind", req.Kind, "namespace", req.Namespace, "name", req.Name, "uid", req.UID,
 		"operation", req.Operation, "user info", req.UserInfo)
-	l.Info("AdmissionReview")
```

Comment:
If we have AdmissionReview printed in every case why not just leave it here? Is it only because of the kafkaCluster case having the if in the beginning?

Reply:
Because if it would be there, then in every case this info would be written in the operator log, but now there is only one exception here: https://github.com/banzaicloud/koperator/blob/3c935f3219cd49cf12f3d8294b6810946ec75219/pkg/webhook/request.go#L60 . This means when the operator change the kafkacluster, we does not validate and write "AdmissionReview", because it happens many times. It pollutes the log unnecessarily.